website/src/theme/CodeBlock/Content/styles.module.css

```
@@ -0,0 +1,23 @@
+.codeBlock {
+  --ifm-pre-background: var(--prism-background-color);
+  margin: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)